pkg/v16/resource/chartoperator/update.go

```diff
@@ -84,6 +84,7 @@ func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange inte
 				updateState.ReleaseName,
 				tarballPath,
 				helm.UpdateValueOverrides(b),
+				helm.UpgradeForce(true),
```

Comment:
The last param for `UpdateReleaseFromTarball` is `options ...helm.UpdateOption` so no changes are needed to helmclient which is pretty nice.